src/jaxsim/integrators/variable_step.py

```diff
@@ -481,7 +484,7 @@ def reject_step():
         with self.mutable_context(mutability=Mutability.MUTABLE):
             self.params = params_tf
 
-        return xf
+        return xf, aux_dict
```

Comment:
Can you also update the signature of this method, aligning it with the base class?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/255/commits/0d5fbf76e8920a04f6b981ffb36b46214887155d